src/test/java/core/basesyntax/services/actions/IncreaseHandlerTest.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.services.actions;
+
+import core.basesyntax.db.Storage;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class IncreaseHandlerTest {
+    private Map<String, ActionHandler> actionHandlerMap;
+
+    @Before
+    public void before() {
+        Storage.fruits.clear();
+        actionHandlerMap = new HashMap<>();
+        actionHandlerMap.put("b",new BalanceHandler());
+        actionHandlerMap.put("s",new IncreaseHandler());
+        actionHandlerMap.put("p",new PurchaseHandler());
+        actionHandlerMap.put("r",new IncreaseHandler());
```

Comment:
I think we don't actually need to initialize all handlers and the handler map, if we want to test only one handler - what do you think?